src/Dash/Router/Parser/ParserManager.php

```diff
@@ -20,8 +20,8 @@ class ParserManager extends AbstractPluginManager
     protected $shareByDefault = false;
 
     protected $factories = [
-        'pathsegment'     => 'Dash\Router\Http\Parser\PathSegmentFactory',
-        'hostnamesegment' => 'Dash\Router\Http\Parser\HostnameSegmentFactory',
+        'pathsegment'     => Dash\Router\Parser\PathSegmentFactory::class,
+        'hostnamesegment' => Dash\Router\Parser\HostnameSegmentFactory::class,
```

Comment:
Actually we already are in the Parser namespace, so I can just reference the class name without namespace here.